test/protocol_test.py

```diff
@@ -3612,6 +3612,7 @@ def test_good_args(self):
             },
         )
         assert len(self.p.instructions) == 1
+        assert self.t1.well(0).volume == Unit(0, "uL")
```

Comment:
Maybe do a transfer to that well prior to evaporation method to make sure that the volume gets reset to zero.